packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -1864,12 +1890,8 @@ abstract class MultiSelectableSelectionContainerDelegate extends SelectionContai
     if (selections.isEmpty) {
       return null;
     }
-    final StringBuffer buffer = StringBuffer();
-    for (final SelectedContent selection in selections) {
-      buffer.write(selection.plainText);
-    }
     return SelectedContent(
-      plainText: buffer.toString(),
+      plainText: selections.map((SelectedContent e) => e.plainText).join(separator),
```

Comment:
I wonder why the StringBuffer approach was used in the first place, is it any more performant?

Reply:
The `join` call uses a `StringBuffer` internally